tests/functional/endpoint/test_endpoint_search.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import annotations
+
+import random
+import typing as t
+import uuid
+
+import pytest
+from globus_sdk._testing import RegisteredResponse, get_last_request
+
+
+def _make_mapped_collection_search_result(
+    collection_id: str,
+    endpoint_id: str,
+    *,
+    display_name: str = "dummy result",
+    authentication_timeout_mins: int = 15840,
+) -> dict[str, t.Any]:
```

Comment:
Providing `display_name` and `authentication_timeout_mins` defaults here means that it's incumbent on fixtures to either know these values and fill them in as `metadata`, or to call this method with their own values and remember to fill in the values as `metadata`.

Below, I see that the single response fixture relied on the defaults and didn't fill them in as metadata. As this is the CLI, not the SDK, this is probably mostly fine. However, I want to point out that using default parameters here means that fixtures can fail to provide all of the metadata about the responses, and this is the silent, default outcome.

Reply:
I don't really know how I want to phrase fixtures for the SDK to provide via `_testing`. These sorts of parameters are part of that fuzzy thinking.

I'm going to leave it as-is right now, as I view this helper as an implementation detail for this test module, but I hear what you're saying about the data being implicit where maybe it should be explicit.

Reply:
Eh, now that I'm in the module I'm changing my mind. I didn't have a clear rationale at play here for selecting these fields in this way.
You'll see `authentication_timeout_mins` disappear (it's just a fixed field in the response), and `display_name` become required.